R-packages/evalcast/vignettes/intro-evalcast.Rmd

```diff
@@ -0,0 +1,211 @@
+---
+title: "Introduction to evalcast"
```

Comment:
If you name the vignette the same thing as the package, pkgdown automatically makes a "get started" link in the top menu for that vignette. Your decision if that's nicer: https://pkgdown.r-lib.org/reference/build_articles.html#get-started

Reply:
Thanks! Done.